ci-operator/config/ComplianceAsCode/content/ComplianceAsCode-content-master__4.15.yaml

```diff
@@ -416,6 +425,66 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- always_run: false
+  as: e2e-rosa-ocp4-cis-node
+  steps:
+    cluster_profile: quay-aws
+    env:
+      BYO_OIDC: "true"
+      CHANNEL_GROUP: stable
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
```

Comment:
This is the same size, or larger than the clusters provided through cluster bot.